src/pp.c

```diff
@@ -1517,7 +1517,7 @@ void do_prince_crack(struct db_main *db, char *wordlist, int rules)
   if (!john_main_process)
     warn = 0;
 
-  wordlist = path_expand(wordlist);
+  wordlist = (char*)path_expand(wordlist);
 
```

Comment:
The above will be fixed soon, changing to `do_prince_crack(..., const char *wordlist, ...)`